clang/tools/scan-build-py/lib/libscanbuild/arguments.py

```diff
@@ -38,20 +38,20 @@ def parse_args_for_intercept_build():
         print('Error: --linker-entry conflicts with --no-linker-entry')
         sys.exit(-1)
 
-    if args.parse_build_log and not args.in_root:
-        print('Error: option --in-root needs to be specified, if --build-log is specified')
+    if args.parse_build_log and not args.work_directory:
+        print('Error: option --work-directory needs to be specified, if --build-log is specified')
         sys.exit(-1)
 
-    if not args.parse_build_log and args.in_root:
-        print('Error: option --build-log needs to be specified, if –in-root is specified')
+    if not args.parse_build_log and args.work_directory:
+        print('Error: option --build-log needs to be specified, if -work-directory is specified')
```

Comment:
-work-directory should be given a default value for user.   consider to use the folder of build log file.

Reply:
ok